templates/rust/src/services/service.rs.twig

```diff
@@ -0,0 +1,46 @@
+use crate::client::{Client, ParamType};
+use std::collections::HashMap;
+
+#[derive(Clone)]
+pub struct {{ service.name | caseUcfirst }} {
+  client: Client
+}
+
+impl {{ service.name | caseUcfirst }} {  
+  pub fn new(client: &Client) -> Self {
+    Self {
+      client: client.clone()
+    }
+  }
+  {% for method in service.methods %}
+
+{% if method.description %}
+{{ method.description|dartComment }}
+{% endif %}
+    pub fn {{ method.name | caseSnake }}(&self{% if method.parameters.all|length >= 1 %}, {% endif %}{% for parameter in method.parameters.all %}{{ parameter.name | caseSnake | escapeKeyword}}: {{ parameter.type | typeName }}{% if not loop.last %}, {% endif %}{% endfor %}) -> reqwest::blocking::Response {
+        let path = "{{ method.path|replace({'{': '', '}': ""}) }}"{% for parameter in method.parameters.path %}.replace("{{ parameter.name | caseSnake }}", &{{ parameter.name | caseSnake }}){% endfor %};
+
+    {% if method.headers %}
```

Comment:
Try avoiding indentation for every template syntax from twig. Meaning, the spaces left of `{% if method.headers %}` are still going to be in the generated file. Just remove the spaces so it is just `{% if method.headers %}` in the line 👍🏻 

This needs to be adapted to every line where only twig stuff is in 🙂